src/components/CommentSection.jsx

```diff
@@ -32,6 +49,12 @@ function CommentSection (){
             <button onClick={commentHandler}>Comments - {reviewComments.length}</button>
             {!showComments ? null :             
             <TableContainer>
+                <form onSubmit={postCommentHandler}>
+                    <br/>
+                    <input type='text' onChange={(event)=>setCommentInput(event.target.value)}/>
```

Comment:
To ensure the comment can go over multiple lines 'textarea' might be a better tag than 'input' - it's a better user experience for someone wanting to write longer form text